docs/features/workspace.md

```diff
@@ -0,0 +1,36 @@
+# Workspace
+A workspace is a version controlled folder that contains a workspace.yml file that define the workspace. The file lists the git repositories that compose the workspace.
+
+When a developer wants to work on a project with mrw, they clone the workspace repo, install mrw and run `mrw init` to get all the repositories listed cloned.
+
+The workspace folder is where the folders of the repositories listed in workspace.yml live. These folders are ignored in the workspace's repository to ensure git does not think we are using submodules.
```

Comment:
Yes, good point, maybe add a link to the mrw philosphy doc, if it is not overwhelming the reader ("too much is like not enough" - *Trop c'est comme pas assez*)